build-local.rb

```diff
@@ -55,10 +63,21 @@ def build_image()
         2>&1"
     output = `#{cmd}`
     if $?.success?
-      @spinner.success
+      if @write_success_log
+        @spinner.success(" - see #{@script_dir}/logs/#{@repo_name}_build.log")
+        @putput = @output + output + "\nBuild failed... Unable to retry.\n" 
+        File.write("#{@script_dir}/logs/#{@repo_name}_build.log", "log from command=#{cmd}\n#{output}", mode: "w")
+      else
+        @spinner.success
+      end
       @image_var = "#{@config['image_env_var']}=#{image_name}\n"
       @success = true
+    elsif @retries > 0
+        @output = @output + output + "\nBuild failed... Retrying...\n"
+        @retries = @retries - 1
+        build_image()
     else
+      @putput = @output + output + "\nBuild failed... Unable to retry.\n" 
```

Comment:
Is this a typo @putput instead of @output

Reply:
Fixed